src/components/Features/Account/Account.module.scss

```diff
@@ -0,0 +1,4 @@
+.accountMenu {
+  padding: 8px;
+  width: 432px;
```

Comment:
Also this?

Reply:
also here account menu has special width

Reply:
What about pass `width` to `Menu`?